src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -544,11 +544,10 @@ boolean canRepair(
 
               String msg = "Postponed due to affected hosts already doing repairs";
               repairRunner.updateLastEvent(msg);
-              handlePotentialStuckRepairs(busyHosts, metrics.getNode());
```

Comment:
Despite the function name, this is not the only codepath for stuck repairs, this is during the scheduling of a repair (this is what happens in the `canRepair` function which is the thing we are modifying)